internal/provider/resource_password.go

```diff
@@ -750,15 +759,16 @@ func passwordSchemaV2() tfsdk.Schema {
 				PlanModifiers: []tfsdk.AttributePlanModifier{
 					resource.RequiresReplace(),
 				},
-				Validators: []tfsdk.AttributeValidator{
-					int64validator.AtLeast(1),
-					int64validator.AtLeastSumOf(
-						path.MatchRoot("min_upper"),
-						path.MatchRoot("min_lower"),
-						path.MatchRoot("min_numeric"),
-						path.MatchRoot("min_special"),
-					),
-				},
+				// TODO: Reinstate once passwordSchemaV2 has been updated to use schema.Schema.
```

Comment:
Since the resource state upgrade prior schemas aren't "live" schemas that would participate in `ValidateResourceConfig` or `PlanResourceChange` RPCs, we could also just drop `PlanModifiers` and `Validators` from them rather than worrying about converting this code. 👍